console/lazy_commands.rst

```diff
@@ -36,6 +45,28 @@ method accepts any
 :class:`Symfony\\Component\\Console\\CommandLoader\\CommandLoaderInterface`
 instance so you can use your own implementation.
 
+Another way to do so is to take advantage of ``Symfony\Component\Console\Command\LazyCommand``:
+
+    use App\Command\HeavyCommand;
+    use Symfony\Component\Console\Application;
+    use Symfony\Component\Console\Command\Command;
+    use Symfony\Component\Console\CommandLoader\FactoryCommandLoader;
+
+    // In this case although the command is instantiated, the underlying command factory
+    // will not be executed unless the command is actually executed or one tries to access
+    // to its input definition.
+    $lazyCommand = new LazyCommand(
+        'app:heavy',
+        [],
+        'This is another more complete form of lazy command.',
+        false,
+        static fn (): Command => new HeavyCommand(),
+    );
+
+    $application = new Application();
+    $application->add($lazyCommand);
+    $application->run();
+
```

Comment:
This example conflicts with the statement added above at l13 isn't it? As wrapping commands in `LazyCommand` nstances makes the `ListCommand` respects laziness through dealing with proxies only

Reply:
AFAIK the list command checks for the description, which is why making a command lazy with `app:lazy => static fn () => new HeavyCommand()` does not work for the list command (as it looks for the description).